lib/travis/api/app/endpoint/authorization.rb

```diff
@@ -134,6 +134,7 @@ class Authorization < Endpoint
         def update_first_login(user)
           unless user.first_logged_in_at
             user.update_attributes(first_logged_in_at: Time.now)
+            user.create_initial_subscription
```

Comment:
I just realized that this method should not be called in `.org` env

Reply:
Absolutely right! Thank you @pavel-d for noticing it.